realm/realm-library/src/objectServer/java/io/realm/mongodb/auth/GoogleAuthType.java

```diff
@@ -0,0 +1,6 @@
+package io.realm.mongodb.auth;
+
+public enum GoogleAuthType {
```

Comment:
We need public API docs for this, including each enum type + missing license headers